interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -2006,34 +1997,60 @@ public void performSoakTest(
       int soakIterations,
       int maxFailures,
       int maxAcceptablePerIterationLatencyMs,
+      int minTimeMsBetweenRpcs,
       int overallTimeoutSeconds)
       throws Exception {
     int iterationsDone = 0;
     int totalFailures = 0;
     Histogram latencies = new Histogram(4 /* number of significant value digits */);
     long startNs = System.nanoTime();
+    ManagedChannel soakChannel = createChannel();
+    TestServiceGrpc.TestServiceBlockingStub soakStub = TestServiceGrpc
+        .newBlockingStub(soakChannel)
+        .withInterceptors(recordClientCallInterceptor(clientCallCapture));
+    soakStub = soakStub.withInterceptors(recordClientCallInterceptor(clientCallCapture));
```

Comment:
Looks like you accidentally duplicated this line.

Reply:
whoops fixed